react-search-component/my-app/src/SearchableList Components/SearchBar.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChangeEvent } from 'react';
+
+type SearchBarProps = {
+  onSearch: (searchText: string) => void;
+};
+
+export default function SearchBar({ onSearch }: SearchBarProps) {
+  function handleOnChange(event: ChangeEvent<HTMLInputElement>) {
```

Comment:
Usually the name would be just `handleChange`. I think of it this way:
"Change" is the event name; "onChange" is the property used to pass an event handler for the Change event; "handleChange" is the event handler.

Reply:
updated